pythran/pythonic/numpy/fft/c2c.hpp

```diff
@@ -0,0 +1,122 @@
+#ifndef PYTHONIC_NUMPY_FFT_C2C_HPP
+#define PYTHONIC_NUMPY_FFT_C2C_HPP
+
+#include "pythonic/include/numpy/fft/c2c.hpp"
+#include "pythonic/utils/functor.hpp"
+#include "pythonic/include/utils/array_helper.hpp"
+#include "pythonic/types/ndarray.hpp"
+#include "pythonic/builtins/None.hpp"
+#include "pythonic/numpy/concatenate.hpp"
+#include "pythonic/numpy/zeros.hpp"
+
+
+#include <array>
+#include <cstring>
+#include <cmath>
+
+#include "pythonic/numpy/fft/pocketfft.hpp"
+
+PYTHONIC_NS_BEGIN
+
+namespace numpy
+{
+  namespace fft
+  {
+    using pocketfft::stride_t;
+    using pocketfft::shape_t;
+    using ldbl_t = typename std::conditional<sizeof(long double)==sizeof(double), double, long double>::type;
+
+    template<typename T> T norm_fct(int inorm, size_t N)
+    {
+        if (inorm==0) return T(1);
```

Comment:
I assume the cast is not needed here, because it's already part of the return type.

Reply:
Could be a `switch` btw

Reply:
Fixed